app/services/admin/participants/search.rb

```diff
@@ -40,11 +40,9 @@ def search_conditions
   end
 
   def type_conditions
-    if type.present?
-      ParticipantProfile.where(type:)
-    else
-      ParticipantProfile.all
-    end
+    scope = ParticipantProfile.all
+    scope = scope.merge(ParticipantProfile.ecf) if NpqApiEndpoint.disabled?
```

Comment:
The `merge` confused me for a second here; would it be more readable without it? something like:

```
scope = NpqApiEndpoint.disabled? ? ParticipantProfile.ecf : ParticipantProfile.all
```